language/compiler/bytecode-source-map/src/source_map.rs

```diff
@@ -255,6 +255,13 @@ impl<Location: Clone + Eq> FunctionSourceMap<Location> {
             self.add_type_parameter((name, default_loc.clone()))
         }
 
+        // Generate names for each parameter
+        let params = module.signature_at(function_handle.parameters);
+        for i in 0..params.0.len() {
+            let name = format!("Arg{}", i);
```

Comment:
Maybe lowercase `arg` instead here?

Reply:
Not sure. I was going for consistency with the generated type parameter names, which are "Ty0", "Ty1", etc. WDYT?

Reply:
Hmm good point. I was thinking lowercase since arguments should be lowercase, but I'm fine keeping this as-is as well.

Reply:
It's easy to change later if people prefer lowercase.... I'll stick with this for now